java/ql/test/experimental/configured-flow/test-remote.ext.yml

```diff
@@ -0,0 +1,7 @@
+extensions:
```

Comment:
Nice. I'd forgotten that extensions are automatically included if they are named the same as the test.